css/components/_verse.css

```diff
@@ -9,7 +9,7 @@
 	width: 100%;
```

Comment:
Yeah, I didn't do anything other than adjust the card class to point to the updated class (I'm sure there are more issues / things that are pointing to a now incorrect selector). 

The css will definitely still need work in another PR. Just wanted to get the structure out there.